src/drive/web/modules/drive/Toolbar/components/Scanner.jsx

```diff
@@ -223,17 +232,19 @@ class Scanner extends React.Component {
     } = this.state
     if (loadingScreen) {
       return (
-        <Overlay>
-          <div className="u-bg-black u-mih-100" />
-        </Overlay>
+        <Modal mobileFullscreen closable={false}>
+          <Overlay>
+            <div className="u-bg-black u-mih-100" />
+          </Overlay>
+        </Modal>
```

Comment:
I'm getting more and more confused by this. The modal already ships with an overlay, why do we want a overlay -> modal -> overlay -> black content ?

Reply:
Oh you right! At the begining I only had an Overlay. But I needed a modal, so I only move the Overlay to the Modal. 

It appears that <Modal mobileFullscreen closable={false} className="u-bg-black u-mih-100" /> do the exact same thing I think !